### PR TITLE
Allign with PR change generated from the originating upstream project

### PR DESCRIPTION
**Pull request by crolarlibertyva**
This aligns with the changes in https://github.com/auth0/passport-wsfed-saml2/pull/143.

### CODE REVIEW

File: lib/passport-wsfed-saml2/strategy.js [hunk 1]
```diff
@@ -202,8 +202,7 @@ WsFedSaml2Strategy.prototype.authenticate = function (req, opts) {
   }
 
   function executeSamlp(req) {
-    if (req.body && ((req.method === 'POST' && req.body.SAMLResponse) ||
-                     (req.method === 'GET' && req.query.SAMLResponse))) {
+    if (req.body && req.method === 'POST' && req.body.SAMLResponse) {
```
Comment (beaugrantham-va) [inline lib/passport-wsfed-saml2/strategy.js:205]: This was added into our fork for a specific reason. I don't think we can remove this.

**Comment by beaugrantham-va** [comment]
Can this be updated with the actual changes that were merged upstream?

- [x] `https://github.com/auth0/passport-wsfed-saml2/commit/881fef82ea6887447594f5d79a233f1c0a3d357e`

- [x] `https://github.com/auth0/passport-wsfed-saml2/commit/dcfcc475995dfb3b6e5e32c0c3149b388d1c3698`

File: lib/passport-wsfed-saml2/strategy.js (hunk 1 above)
Comment (crolarlibertyva) [reply lib/passport-wsfed-saml2/strategy.js]: I've reverted for now, but I did repeat the debug runs to confirm that the req.method hasn't yet been a 'GET' in my testing.

**Comment by beaugrantham-va** [comment]
Closing in favor of API-5707 resolution.

**Pull request closed without merging**